lib/tasks/valid_test_data_generator/completed_mentor_generator.rb

```diff
@@ -0,0 +1,233 @@
+# frozen_string_literal: true
+
+require "tasks/school_urn_generator"
+require "tasks/trn_generator"
+require "active_support/testing/time_helpers"
+
+module ValidTestDataGenerator
+  class CompletedMentorGenerator
+    include ActiveSupport::Testing::TimeHelpers
+
+    class << self
+      def call(name:, cohort: Cohort.current, total_completed_mentors: 30)
+        new(name:, cohort:).call(total_completed_mentors:)
+      end
+    end
+
+    def call(total_completed_mentors:)
+      school = lead_provider.schools.order("RANDOM()").first
```

Comment:
`school = lead_provider.schools.joins(:school_cohorts).where(school_cohorts: { induction_programme_choice: "full_induction_programme" }).order("RANDOM()").first`

Think the generator is a bit flakey as I haven't got the same issue as Ross when I ran the first time.. but am getting it now.. that might fixes the issue I guess

Reply:
Added the fix.👍